modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/css/main.scss

```diff
@@ -769,7 +769,33 @@ $productMenuWidth: 320px;
 	.upload-drop-active &,
 	.upload-drop-intent & {
 		.document-container {
-			background-color: #eef4ff;
+			background-color: $primary-l3;
+		}
+	}
+
+	.upload-drop-active & {
+		.document-container {
+			border-radius: 2px;
+			box-shadow: 0 0 0 2px $primary-l2 !important;
+
+			.active-area {
+				background-color: transparent !important;
```

Comment:
I don't like the `!important` here, because if it is defined in a theme, it meanse it is configurable and we should not override it... but, if you are really sure that we need that background (according to the designs), then we can leave it.

Reply:
Yep, I talked with Maria about the list and table view too we want primary-l3 as a background.
We can remove/update the theme part but I don't know the implications, is not document-library specific.
Do you have a better way to achieve it?

Reply:
We can remove/update the theme part